### PR TITLE
python3Packages.python3-msrplib: init at 0.21.1

### PR DESCRIPTION
**Pull request by charlieshanley**
###### Motivation for this change

Split from #136924. This adds a dependency of blink-qt and sipclients3, which I intend to add.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by charlieshanley** [comment]
Result of `nixpkgs-review pr 139557` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.python3-msrplib</li>
    <li>python39Packages.python3-msrplib</li>
  </ul>
</details>

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-msrplib";
+    rev = "5bd069620d436d5a65e1c369e43cc6b88857fb9e"; # No tag for latest version
+    sha256 = "sha256-z0gF/oQW/h3qiCL1cFWBPK7JYzLCNAD7/dg7HfY4rig=";
+  };
+
+  propagatedBuildInputs = [ python3-application python3-gnutls python3-eventlib twisted zope_interface ];
```
Comment (risicle) [inline pkgs/development/python-modules/python3-msrplib/default.nix:17]: This should probably be spilled to one dep per line.

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
```
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/development/python-modules/python3-msrplib/default.nix:18]: It is required actually.

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.21.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-msrplib";
+    rev = "5bd069620d436d5a65e1c369e43cc6b88857fb9e"; # No tag for latest version
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python3-msrplib/default.nix:13]: Please at least open an issue about upstream.

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, python3-application
+, python3-gnutls, python3-eventlib, twisted, zope_interface }:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.21.1";
+
+  disabled = !isPy3k;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python3-msrplib/default.nix:8]: Please use pythonOlder instead.

**Pull request closed without merging**